src/core/server/saved_objects/saved_objects_config.ts

```diff
@@ -30,3 +30,29 @@ export const config = {
     skip: schema.boolean({ defaultValue: false }),
   }),
 };
+
+export type SavedObjectsConfigType = TypeOf<typeof savedObjectsConfig.schema>;
+
+export const savedObjectsConfig = {
+  path: 'savedObjects',
+  schema: schema.object({
+    maxImportPayloadBytes: schema.number({ defaultValue: 10485760 }),
+    maxImportExportSize: schema.number({ defaultValue: 10000 }),
+  }),
+};
```

Comment:
For the import/export routes, I now need the 'real' savedObject config. What we called that previously was the config for the `migrations` config namespace. I created the `SavedObjectConfig` class to 'merge' them. I kept the migrations properties as an underlying object to avoid breaking every signature in the SO migrator code, and also because I think this isolation makes sense.

Reply:
not a blocker, just curious: Do we want to move `migrations` --> `savedObjects.migration` in the future? Will we have migrations fields in v8 with saved objects v2? @rudolf